ansible/library/shell_cmds.py

```diff
@@ -151,8 +154,11 @@ def main():
     )
 
     if output['failed']:
+        for failed_cmd in failed_cmds:
```

Comment:
can you give an example of the output after this change

Reply:
hi @arlakshm updated in PR description